hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -527,7 +529,10 @@ int SaraNcpClient::dataChannelWrite(int id, const uint8_t* data, size_t size) {
         }
     }
 
-    int err = muxer_.writeChannel(UBLOX_NCP_PPP_CHANNEL, data, size);
+    int err = -1;
```

Comment:
This should be `0` by default, right? Otherwise we'll fall into `disable()` below on line 544.

Reply:
Yeah, I'll change to `int err = gsm0710::GSM0710_ERROR_NONE;`